src/block/block_ckpt.c

```diff
@@ -72,8 +72,8 @@ __wt_block_checkpoint_load(WT_SESSION_IMPL *session, WT_BLOCK *block, const uint
         live_open = block->live_open;
         block->live_open = true;
         __wt_spin_unlock(session, &block->live_lock);
-        WT_ERR_ASSERT(
-          session, live_open == false, EBUSY, "%s: attempt to re-open live file", block->name);
+        WT_ERR_ASSERT(session, WT_DIAG_CONCURRENT_ACCESS, live_open == false, EBUSY,
```

Comment:
Need to think more about the diag category "concurrent_access", it did not feel immediately obvious to me for this assert.